### PR TITLE
Bump to avalanchego 1.7.11

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

File: local/node.go [hunk 1]
```diff
@@ -123,16 +126,19 @@ func (node *localNode) AttachPeer(ctx context.Context, router router.InboundHand
 	if err != nil {
 		return nil, err
 	}
-	ip := avago_utils.IPDesc{
+	ip := ips.IPPort{
 		IP:   net.IPv6zero,
 		Port: 0,
 	}
+	resourceTracker, err := tracker.NewResourceTracker(prometheus.NewRegistry(), resource.NoUsage, meter.ContinuousFactory{}, 10*time.Second)
```
Comment (dasconnor) [inline local/node.go:133]: nit: line length

File: network/config.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func NewAvalancheGoGenesis(
 	}
 
 	// Address that controls stake doesn't matter -- generate it randomly
-	genesisVdrStakeAddr, _ := formatting.FormatAddress("X", constants.GetHRP(networkID), ids.GenerateTestShortID().Bytes())
+	genesisVdrStakeAddr, _ := address.Format("X", constants.GetHRP(networkID), ids.GenerateTestShortID().Bytes())
```
Comment (dasconnor) [inline network/config.go:115]: nit: line length

**Review by dasconnor — APPROVED** [review]
LGTM

**Review by holisticode — APPROVED** [review]

File: local/node.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (felipemadero) [reply local/node.go]: done

**Pull request merged**